api/views/mixins.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import request, serializers
+from rest_framework.permissions import SAFE_METHODS
+
+
+class ReadWriteSerializersMixin:
```

Comment:
That's interesting - isn't anything available out of the box in DRF?

Reply:
Nope. Other option is to redefine it in every viewset, or write common method to, for example, get serializer class by action (list/retrieve/etc.). And I'm not sure we need it now, because we have only two cases - read and write.